ui/src/app/edge/live/energymonitor/chart/section/consumptionsection.component.ts

```diff
@@ -35,10 +64,19 @@ export class ConsumptionSectionComponent extends AbstractSection {
     }
 
     protected _updateCurrentData(sum: DefaultTypes.Summary): void {
+        let arrowIndicate: number;
+        if (sum.consumption.activePower > 49) {
```

Comment:
"49" vermutlich, weil das noch auf "0" gerundet wird, oder? Bitte einen Kommentar dazu, sonst versteht das in zwei Wochen niemand mehr.